content/releases/prestashop80.md

```diff
@@ -0,0 +1,504 @@
+                            </p>
+                            <p>
+                                <strong>Customize your distribution.</strong> During the shop’s installation, any modules and themes present in the shop’s modules and themes directories will be installed by default, unless the user chooses otherwise. This allows integrators to create customized PrestaShop builds with (or without) the modules and themes of your choosing.
+                            </p>
+                        </div>
+                        <div class="col-md-6">
+                            <p>
+                                <strong>Find out exactly what went wrong if the installation fails.</strong> If debug mode is on, a full stack trace will be displayed after something goes wrong during the installation process. This will definitely help developers pinpoint the source of the problem.
+                            </p>
+                            <p>
+                                <strong>Extend the installation process.</strong> Modules now include a callback <code>postInstall()</code>, allowing them to executed code after the shop’s installation is finished. This provides the developers with new possibilities, like add extra steps to the installation, preload assets, or preprocess data.
+                            </p>
+                        </div>
+                    <div class="row">
+                        <div class="col-md-6">
+                            <p>
+                                <img loading="lazy" class="image-with-shadow" src="/releases/images/ps8/installer_select_modules.jpeg">
+                            </p>
+                        </div>
+                        <div class="col-md-6">
+                            <p>
+                                <img loading="lazy" class="image-with-shadow" src="/releases/images/ps8/installer_errors.jpeg">
+                            </p>
+                        </div>
+                    </div>
+                </div>
+            </section>
+            <section class="section" id="webservice">
+                <div class="container">
+                    <div class="row">
+                        <div class="col-md-4">
+                            <h2 class="section-title">Webservice</h2>
+                            <p>
+                                <strong>The status and URL to your shop’s webservice is now displayed</strong> at the top of the page in <em>Advanced parameters > Webservice</em>. This should be useful to new users and help them troubleshoot common problems.
+                            </p>
+                            <p><strong>*It is now possible to perform partial updates on webservice endpoints using the PATCH method.</strong> This allows integrations to update only part of an entity instead of all the fields at once. <a href="https://github.com/PrestaShop/PrestaShop/pull/27952">Learn more about this change here</a>
```

Comment:
```suggestion
                            <p><strong>*It is now possible to perform partial updates on webservice endpoints using the PATCH method.</strong> This allows integrations to update only part of an entity instead of all the fields at once. <a href="https://github.com/PrestaShop/PrestaShop/pull/27952">Learn more about this change here</a>.
```